test/OriginalTokenBridge.test.js

```diff
@@ -307,6 +307,51 @@ describe("OriginalTokenBridge", () => {
         })
     })
 
+    describe("sets withdrawal fee", () => {
+        const withdrawalFeeBps = 100
+        it("reverts when called by non owner", async () => {
+            await expect(originalTokenBridge.connect(user).setWithdrawalFeeBps(withdrawalFeeBps)).to.be.revertedWith("Ownable: caller is not the owner")
+        })
+
+        it("sets withdrawal fee", async () => {
```

Comment:
please add a test to to cover setting value  higher then TOTAL_BPS

Reply:
@leonprou added